notebooks/09 - Suitability_Analysis.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
Hmm I wonder why Git can't tell that this notebook was moved, and rather thinks it was deleted and re-added

Reply:
yeah not sure..